### ISSUE TITLE
`solutes.water.nameProperty` does not migrate

### ISSUE THREAD

**Issue by zepumph** [phetsims/ph-scale#267]
@Nancy-Salpepi originally reported this in https://github.com/phetsims/phet-io-sim-specific/issues/17#issuecomment-1382824832:

This still seems to not work for water in https://github.com/phetsims/qa/issues/872

1. In Studio version 1.5, change `phScale.global.model.solutes.water.nameProperty` to "H2O"
2. Save file 
3. Load into Studio version 1.6 --it says "Water"

Also in studio version 1.5, if you make this additional change:
1. Change phScale.macroScreen.view.waterFaucetNode.waterLabelNode.textProperty to "spit"
2. Save file
3. Load into Studio version 1.6--it says "Water"
<img width="417" alt="Screenshot 2023-01-14 at 10 16 50 AM" src="https://user-images.githubusercontent.com/87318828/212479809-afb316c0-ec78-4914-949a-da28cad26009.png">

<img width="551" alt="Screenshot 2023-01-14 at 10 34 23 AM" src="https://user-images.githubusercontent.com/87318828/212480092-bd4f4dcd-d8e6-41ff-a347-92d148474c9e.png">

**Comment by Nancy-Salpepi** [comment]
Right. I think it is because water appears in two places and there used to be 2 different strings but now there is only one.

**Comment by pixelzoom** [comment]
This blocks publication of the next RC.

**Comment by pixelzoom** [comment]
Since this problem is specific to `solutes.water.nameProperty`, I changed the issue title.

**Comment by pixelzoom** [comment]
@Nancy-Salpepi said:
> Right. I think it is because water appears in two places--on the faucet and in the combobox-- and there used to be 2 different strings but now there is only one.

While "Water" does indeed appear in 2 places in the UI, there is (and always has been) only 1 entry in the strings.json file:

```
  "choice.water": {
    "value": "Water"
  },
```

The difference is that `phScale.global.model.solutes.water.nameProperty` was instrumented in 1.5, while the text for `phScale.microScreen.view.waterFaucetNode.waterLabelNode` was not.

**Comment by pixelzoom** [comment]
These rules also look suspect:

```
    // These have been replaced with PHScaleStrings.waterStringProperty.
    // NOTE! These must be done before rename of waterLabelNode below.
    new TandemFragmentRenamed( 'phScale.macroScreen.view.waterFaucetNode.waterLabelNode.stringProperty',
      'phScale.general.model.strings.phScale.choice.waterStringProperty' ),
    new TandemFragmentRenamed( 'phScale.microScreen.view.waterFaucetNode.waterLabelNode.stringProperty',
      'phScale.general.model.strings.phScale.choice.waterStringProperty' ),
```

I don't believe that scenery.Text had `stringProperty` in 1.5, that was a relatively-recent rename. Wasn't it something like `textProperty`?

**Comment by zepumph** [comment]
Because of a logic bug fix in https://github.com/phetsims/phet-io-wrappers/issues/480, and the reordering in https://github.com/phetsims/phet-io-sim-specific/commit/99215da33b4f4ca571d90c74d1d665a5b2572780, the value for waterStringProperty will now be decided first and foremost by the name from the solutes. Please note that https://github.com/phetsims/phet-io/issues/1903#issuecomment-1386300547 will prohibit testing this on master for a time, so I'll mark this on hold, assigned to myself until we can fix that.

**Comment by samreid** [comment]
We need commits from:

- [ ] https://github.com/phetsims/phet-io-wrappers/issues/480

**Comment by zepumph** [comment]
Ready to cherry pick here, don't forget about https://github.com/phetsims/phet-io-wrappers/issues/480

**Comment by samreid** [comment]
@zepumph and I cherry picked and verified this. Would be good for QA to verify it in RC.2.

**Comment by zepumph** [comment]
I confirmed in ph-scale and basics 1.6.0-rc.2 that the value of the water customization comes from the solutes. Ready to close after QA verification.

**Comment by Nancy-Salpepi** [comment]
This looks good in rc.3 for pH Scale and pH Scale basics using the steps in https://github.com/phetsims/ph-scale/issues/267#issuecomment-1407168190 and using the old method too (saving in old studio version and loading in new). 
Closing.
